src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -53,6 +53,10 @@ public static bool CanBeConst(this TypeSymbol typeSymbol)
         /// T where T : IComparable => true
         /// T where T : IComparable? => true
         /// </summary>
+        /// <remarks>
```

Comment:
The comment above is wrong after this change in two cases:

```
T => false
T where default => false
```

Reply:
The behavior of the method is unchanged. However the name of the method is now misleading since it essentially refers to C#8 behavior.

---
In reply to: [456082563](https://github.com/dotnet/roslyn/pull/45993#discussion_r456082563) [](ancestors = 456082563)

Reply:
Renamed to `IsTypeParameterDisallowingAnnotationInCSharp8`.

---
In reply to: [457835131](https://github.com/dotnet/roslyn/pull/45993#discussion_r457835131) [](ancestors = 457835131)